src/scheduler.js

```diff
@@ -112,6 +113,18 @@ class Scheduler extends events.EventEmitter {
         taskId: hook.nextTaskId,
         time: new Date(),
       };
+
+      try {
+        await this.LastFire.create({
+          hookGroupId: hook.hookGroupId,
+          hookId: hook.hookId,
+          taskId: lastFire.taskId,
+          taskCreateTime: lastFire.time,
```

Comment:
we'll want to capture the result here, too.

Reply:
Here I considered the table for storing only success results. So shall we consider storing both success and error.

Reply:
Yes, I think both are important when debugging.